week3/src/components/Header.jsx

```diff
@@ -0,0 +1,72 @@
+// styled API를 사용하여 스타일링
+import styled from "styled-components";
+import PropTypes from "prop-types";
+import { useEffect, useState } from "react";
+import "./SingleCard.css";
+
+const Header = ({ counter, length }) => {
+  const [animate, setAnimate] = useState(false);
+  useEffect(() => {
+    if (counter != 0) {
+      setAnimate(true);
+      setTimeout(() => setAnimate(false), 1000);
+    }
+  }, [counter]);
+  return (
+    <>
+      <ChokiPokiHeader>
+        <strong> 💗 쵸키랑 푸키를 맞춰주세요! 💗</strong>
+        <AnswerCounter className={animate ? "scale" : ""}>
+          <strong>
+            {counter} / {length}
+          </strong>
+        </AnswerCounter>
+      </ChokiPokiHeader>
+    </>
+  );
+};
+
+Header.propTypes = {
+  counter: PropTypes.number,
+  length: PropTypes.number,
```

Comment:
필수로 들어가야 하는 prop이니까 `.isRequired`를 추가해주는것도 좋겠당!!
```suggestion
  counter: PropTypes.number.isRequired,
  length: PropTypes.number.isRequired,
```